src/Relude/Applicative.hs

```diff
@@ -30,6 +32,17 @@ import Control.Applicative (Alternative (..), Applicative (..), Const (..), ZipL
 pass :: Applicative f => f ()
 pass = pure ()
 
+{- | For chaining applicative operations in forward applications using '(&)'
```

Comment:
I think documentation can mention that this function is related to the `<**>` operator.
```suggestion
{- | For chaining applicative operations in forward applications using '(&)'.
Named version of '<**>' operator.
```

Reply:
Perhaps mentioning that this is a flipped version of `<*>` would be helpful